frontend/src/dashboard.js

```diff
@@ -64,6 +66,7 @@ const Dashboard = () => {
   const [selectInputValue, setSelectInputValue] = useState('');
   const [selectFilterValue, setSelectFilterValue] = useState('');
   const selectInputRef = useRef();
+  const [loadKey, setLoadKey] = useState(nanoid(6));
```

Comment:
This is not a very efficient way to do this, at all, but given the scale of the number of widgets on dashboards, I'm going to go with this for now.

I've worked on making a more targeted update to widgets on edit, but I think it will take significant changes that will go well beyond the intent of this PR. I've been iterating with Sonnet and am not that close to a reasonable solution.